### PR TITLE
ISDB-238:

### PR DESCRIPTION
**Pull request by Murshadhaque**
Update case

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by rahulsanjava19 — APPROVED** [review]
LGTM

File: src/main/app/case/CaseApi.ts [hunk 1]
```diff
@@ -33,12 +33,8 @@ export class CaseApi {
    * @param formData
    * @returns
    */
-  public async getOrCreateCaseNew(
-    req: AppRequest,
-    userDetails: UserDetails,
-    formData: Partial<Case>
-  ): Promise<CaseWithId> {
-    return this.createCaseNew(req, userDetails, formData);
+  public async getOrCreateCaseNew(req: AppRequest, userDetails: UserDetails): Promise<CaseWithId> {
```
Comment (parvezhmcts) [inline src/main/app/case/CaseApi.ts:36]: do we actually need this method? can't we directly call createCaseNew from controller?

File: src/main/app/case/CaseApi.ts [hunk 2]
```diff
@@ -56,14 +52,58 @@ export class CaseApi {
...
+   * @param  formData
    * @returns
    */
-
-  public async createCaseNew(req: AppRequest, userDetails: UserDetails, formData: Partial<Case>): Promise<any> {
-    //***** this part need to be implemented on creating a new case  */
-    console.log({ case: req.session.userCase, userDetails, formData });
-    return req.session.userCase;
+  public async updateCase(req: AppRequest, userDetails: UserDetails): Promise<any> {
+    console.log(CaseApi.name + 'Calling backend api update case');
```
Comment (parvezhmcts) [inline src/main/app/case/CaseApi.ts:59]: we should delete console log before committing.

**Review by parvezhmcts — APPROVED** [review]
Approved.

**Pull request merged**